docs/changehistory/NextVersion.md

```diff
@@ -2,11 +2,34 @@
 
 Table of contents:
 
-- [@itwin/components-react](#itwincomponents-react)
+- [@itwin/appui-react](#itwinappui-react)
   - [Deprecations](#deprecations)
+  - [Additions](#additions)
+  - [Changes](#changes)
+- [@itwin/components-react](#itwincomponents-react)
+  - [Deprecations](#deprecations-1)
 - [@itwin/core-react](#itwincore-react)
   - [Fixes](#fixes)
 
+## @itwin/appui-react
+
+### Deprecations
+
+- Deprecated `StatusBar` component in favor of `StatusBarComposer`. [#848](https://github.com/iTwin/appui/pull/848)
+- Deprecated `StatusBar` namespace. Please use the components directly. [#848](https://github.com/iTwin/appui/pull/848)
+  - Deprecated `StatusBar.Popup` and `StatusBarPopup` in favor of `StatusBarPopover`.
+  - Deprecated `StatusBar.Field` and `StatusBarField` in favor of [iTwinUI Button](https://itwinui.bentley.com/docs/button).
+- Deprecated `StatusBarIndicator` in favor of [iTwinUI Button](https://itwinui.bentley.com/docs/button) (or other components) and AppUI `StatusBarPopup`. [#848](https://github.com/iTwin/appui/pull/848)
+- Deprecated `StatusBarLabelIndicator` in favor of [iTwinUI Label](https://itwinui.bentley.com/docs/typography#label) and AppUI `Icon`. [#848](https://github.com/iTwin/appui/pull/848)
```

Comment:
Let's not advocate the `AppUI` `Icon` which is mainly for handling those outdated `IconSpec` types internally. Instead, just point to iTwinUI react package components. (applies to component documentation as well)

Reply:
I suggested using `AppUI` `Icon` for two reasons that are closely related. The first one is that the deprecated `StatusBarLabelIndicator` used `AppUI` `Icon` and the second one is that our `Icon` adds some additional styling. For example, if we use iTwinUI `Icon` or just svgs directly it will have a muted color which is not what we want.